### PR TITLE
tool: change the name from toolbox to migrator

### PR DESCRIPTION
**Pull request by subhamkrai**
this commit changes the name from toolbox to migrator
as the RBAC that was added can be removed after the use
of the tool. And, users don't get confused with the default
toolbox.

Signed-off-by: subhamkrai <srai@redhat.com>

### CODE REVIEW

**Comment by subhamkrai** [comment]
FYI, this PR is on top of #20

**Comment by subhamkrai** [comment]
CC @travisn

File: tests/manifests/migrator.yaml
Comment (travisn) [inline tests/manifests/migrator.yaml:25]: There is no more `tini` in the rook image, see https://github.com/rook/rook/pull/8805 for similar changes we need here as well. Could be a separate PR though...

Comment (subhamkrai) [reply tests/manifests/migrator.yaml:25]: Since,`tini` remove PR merged in master and flex is removed from the master. So, I was using all the required files from rook v1.7.x only. But, I can update in follow up PR soon.

Comment (subhamkrai) [reply tests/manifests/migrator.yaml:25]: #26

Comment (travisn) [reply tests/manifests/migrator.yaml:25]: Yeah I realized on #26 that we don't need that change

**Review by travisn — COMMENTED** [review]
LGTM after #20 is merged and this is rebased

**Comment by subhamkrai** [comment]
> LGTM after #20 is merged and this is rebased

done.

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**
